src/test/java/uk/gov/companieshouse/extensions/api/attachments/file/FileTransferApiClientUnitTest.java

```diff
@@ -112,10 +107,9 @@ public void testUpload_GenericExceptionResponse() {
 
         when(restTemplate.postForEntity(eq(DUMMY_URL), any(), eq(FileTransferApiResponse.class))).thenThrow(exception);
 
-        expectedException.expect(RestClientException.class);
-        expectedException.expectMessage(exception.getMessage());
-
-        fileTransferApiClient.upload(file);
+        RestClientException thrown = assertThrows(RestClientException.class, () ->
+            fileTransferApiClient.upload(file));
+        assertEquals(thrown.getMessage(), thrown.getMessage());
```

Comment:
assertEquals(thrown.getMessage(), thrown.getMessage()); will never be false, it's the same as saying does a == a
we need to check that the message in the thrown exception is what is expected
assertEquals(EXCEPTION_MESSAGE, thrown.getMessage());